src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/signup/SignUpPageObject.java

```diff
@@ -49,7 +49,7 @@ public SignUpPageObject(WebDriver driver) {
   private WebElement birthDayField;
   @FindBy(css = "select[name='birthyear']")
   private WebElement birthYearField;
-  @FindBy(css = "#recaptcha_response_field")
+  @FindBy(css = ".error-msg")
```

Comment:
Why use class `error-msg` here?
Maybe change the element name from `captchaField` then to reflect its purpose.